src/menu/views/load_rom.c

```diff
@@ -362,7 +363,14 @@ static void load (menu_t *menu) {
         case ROM_TV_TYPE_MPAL: menu->boot_params->tv_type = BOOT_TV_TYPE_MPAL; break;
         default: menu->boot_params->tv_type = BOOT_TV_TYPE_PASSTHROUGH; break;
     }
-    menu->boot_params->cheat_list = NULL;
+    cheat_load_err_t cheat_err = load_cheats(menu);
+    if (cheat_err != CHEAT_LOAD_OK) {
+        menu_show_error(menu, cheat_load_convert_error_message(cheat_err));
+        return;
```

Comment:
If cheats are not loaded, are you sure you want to return? that will stop the ROM from loading?!

Reply:
Would it be possible to display the error, but proceed with boot after a timeout ?

Reply:
I can also just set cheat_list to NULL if there is an error, but I wanted to provide the user with feedback in case something happened

Reply:
We need to think about that senario in a bit more depth then.

Reply:
An idea is to parse the cheats not at game launch, but rather when a user enables cheats for their game. This way, feedback can be provided way earlier in the timeline without hindering at the game boot process